src/interface/web/app/components/chatInputArea/chatInputArea.module.css

```diff
@@ -1,4 +1,5 @@
 div.actualInputArea {
     display: grid;
     grid-template-columns: auto 1fr auto auto;
+    max-width: 700px;
```

Comment:
Why is this necessary?

Reply:
It was added in the middle to constrain width when lots of images are added but with the carousel implementatino is not necessary. Removed